src/air_markdown/tags.py

```diff
@@ -65,4 +67,65 @@ def render(self) -> str:
 
 class TailwindTypographyMarkdown(Markdown):
     def wrapper(self, content) -> str:
-        return f'<article class="prose">{content}</article>'
\ No newline at end of file
+        return f'<article class="prose">{content}</article>'
+
+
+class AirHTMLRenderer(HtmlRenderer):
+    def render_block_code(self, token: block_token.BlockCode) -> str:
+        """Render air-live code blocks as the executed output
+        of calling the Air Tag's .render() method.
+
+        For example:
+        ```air-live
+        air.H1("Title")
+        air.P("Paragraph")
+        ```
+        will render as `<h1>Title</h1>\n<p>Paragraph</p>`
+        """
+        template = "<pre><code{attr}>{inner}</code></pre>"
+        if token.language == "air-live":
+            code = token.content.strip()
+            if not code:
+                return ""
+
+            try:
+                module = ast.parse(code)
+                rendered_parts = []
+                local_scope = {}  # Initialize local_scope here
+
+                for node in module.body:
+                    statement_module = ast.Module(body=[node], type_ignores=[])
+                    if isinstance(node, ast.Expr):
+                        # Evaluate expression and render if it's an Air Tag
+                        expr_obj = compile(ast.Expression(body=node.value), "<string>", "eval")
+                        # Ensure local_scope is passed to eval
+                        result = eval(expr_obj, globals(), local_scope)
+                        if isinstance(result, air.Tag):
+                            rendered_parts.append(result.render())
+                    else:
```

Comment:
Are we sure we want to allow inline imports and the `exec` built-in? This turns AirMarkdown into something like Mako or Mighty, which is problematic.

I really like the AST, I'm not a fan of this segment.